lib/govuk_schemas/random_item_generator.rb

```diff
@@ -10,8 +10,10 @@ module GovukSchemas
   #
   # @private
   class RandomItemGenerator
```

Comment:
If you like my suggestion of renaming RandomHelpers then I think this should also be renamed, RandomItemGenerator seems rather ambiguous. Might be clearer as RandomSchemaGenerator.